pkg/tahoe-lafs

```diff
@@ -0,0 +1,38 @@
+[mirrors]
+https://tahoe-lafs.org/downloads/tahoe-lafs-1.13.0.tar.gz
+
+[vars]
+filesize=1710199
+sha512=c8986edae1bea0ac4ecf8f97e0e4ee89ff5024a0ab6039ba21585e9ac2604b7b43bac01d63b91812eef269629b9a5ba99043a30bb12a0ea16c4d13496e24464a
+pkgver=1
+desc="Tahoe-LAFS is a Free and Open decentralized cloud storage system. It distributes your data across multiple servers. Even if some of the servers fail or are taken over by an attacker, the entire file store continues to function correctly, preserving your privacy and security."
```

Comment:
that desc is supposed to be 80 char max, also the convention is to use upper case only for names, like TCP or NFS